src/graphql/helpers/post.js

```diff
@@ -0,0 +1,44 @@
+import rp from 'request-promise-native';
+
+const getFeedParams = ({ user, post }, args, rankBy, filters = {}) => {
+  const userId = user ? user.userId : null;
+
+  return {
+    fields: userId ? post.defaultUserFields : post.defaultAnonymousFields,
+    filters: Object.assign({}, { before: args.latest }, filters),
+    rankBy,
+    userId,
+    page: args.page,
+    pageSize: args.pageSize,
+  };
+};
+
+const splitArrayStr = str => (str ? str.split(',') : null);
```

Comment:
Duplication of code from https://github.com/dailynowco/daily-api/blob/master/src/routes/posts.js#L13.
Maybe you can use the same function in both files

Reply:
I could create an `utils/` directory where I'd put the common functions. Is it okay?

Reply:
Yes, perfect